laravel-app/app/Services/CommentService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Services;
+
+use App\Http\Requests\CommentRequest;
+use App\Models\PostComment;
+use App\Models\UserPost;
+
+class CommentService
+{
+    /**
+     * Logic for storing comment
+     */
+    public function storeComment(CommentRequest $request, UserPost $post): void
+    {
+        if ($user = auth()->user()) {
+            $post->comments()->create([
+                'user_id' => $user->id,
+                'comment' => $request->comment,
+            ]);
+        }
+    }
+
+    /**
+     * Logic for deleting comment
+     */
+    public function deleteComment(int $id): void
+    {
+        if ($comment = PostComment::find($id)) {
+            $comment->delete();
+        }
+    }
+
+    /**
+     * Logic for editing comment
+     */
+    public function editComment(int $id, string $validatedData): void
+    {
+        if ($comment = PostComment::find($id)) {
+            $comment->comment = $validatedData;
+            $comment->save();
+        }
+    }
```

Comment:
Why does this method return void? What if I want to get the updated comment and display it to my controller?

Reply:
Yes sir, will resolve it 🙇